src/utilities/DisplayUtilities.js

```diff
@@ -92,13 +92,13 @@ export const formatTitle = (title, showTitle, loading, error, helpText) => {
     }
 
     if (loading) {
-        return <h5>{`${displayTitle} (loading)`}</h5>;
+        return <h4>{`${displayTitle} (loading)`}</h4>;
```

Comment:
Would these be better using material's typeography components?

Reply:
Yes. As above